docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>One thing that we didn't mention is what coords are used in arviz when the model only defines the dimension name and lets the explicit or implicit size/shape determine the final shape of the RV.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>I think it's fine to omit that detail from here, as this is not so concerned with the InferenceData machinery</p>